src/SFA.DAS.Employer.FrontDoor.Web/Infrastructure/RouteNames.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace SFA.DAS.Employer.FrontDoor.Web.Infrastructure
+{
+    public static class RouteNames
+    {
+        public const string ServiceStartDefault = "default";
```

Comment:
Could we remove the route names that aren't used?

Reply:
Can do, just wasn't sure which we would need so figured would remove the redundant ones later